ydb/core/grpc_services/rpc_rate_limiter_api.cpp

```diff
@@ -66,19 +67,8 @@ class TRateLimiterRequest : public TRpcOperationRequestActor<TDerived, TRequest>
 
         if (resource.has_metering_config()) {
             auto self = static_cast<TDerived*>(this);
-            const auto& userTokenStr = self->Request_->GetSerializedToken();
-            bool allowed = AppData()->AdministrationAllowedSIDs.empty();
-            if (userTokenStr) {
-                NACLib::TUserToken userToken(userTokenStr);
-                for (auto &sid : AppData()->AdministrationAllowedSIDs) {
-                    if (userToken.IsExist(sid)) {
-                        allowed = true;
-                        break;
-                    }
-                }
-            }
 
-            if (!allowed) {
+            if (!IsAdministrator(AppData(), self->Request_->GetSerializedToken())) {
```

Comment:
I believe that `GetSerializedToken()` could be replaced with `GetInternalToken()` that returns `TUserToken` that is already exists in the request context.